transform/mattermost-analytics/models/marts/product/nps/_nps__models.yml

```diff
@@ -37,3 +37,90 @@ models:
       - name: user_email
         description: The email of the user who submitted feedback (optional).
         tags: [pii]
+
+  - name: fct_nps_score
+    description: |
+      This model contains NPS user count metrics on server parameters (server_id, server_version, user_role) sent from Rudderstack and Segment sources.
+      Score is calculated for servers after April 2020, since some servers from Segment reported the user_id's as empty prior.
+      This raises divide by zero error with the NPS score calculation.
+
+    columns:
+      - name: activity_date
+        description: The date the NPS score event was sent. 
+        tests:
+          - not_null
+      - name: server_id
+        description: The ID of the server the NPS score event originated from..
+        tests:
+          - not_null
+      - name: server_version
+        description: The server's version.
+      - name: daily_server_id
+        description: A unique identifier for the nps score generated from server_id, activity_date.
+        tests:
+          - not_null
+          - unique           
+      - name: promoters
```

Comment:
Column names do not match the names in the model.

Reply:
Updated yml file